### ISSUE TITLE
IMPROVEMENTS / bump version

### ISSUE THREAD

**Issue by arosenhagen** [AmpersandHQ/magento-ce-ee-config-corruption-bug#7]
just to keep track of your progress on this - the recent changes are not reflected yet in the patch file.
If those may be breaking changes I would consider to publish PATCH_SUPEE-4755_EE_1.13.1.0_v2.sh within this repo.

**Comment by convenient** [comment]
Hey @arosenhagen 

The reason the newest changes are not included in the SUPEE-4755 patch is that they are not an "official" patch from Magento. 

SUPEE-4755 was released after Magento reviewed this write up. If they read the IMPROVEMENTS.md file and bundle those changes as a new SUPEE- patch file then I'll definitely be adding it to the repo. In the meantime, the changes will have to be made manually.

I would be happy to review any pull requests you have for a file like `PATCH_CUSTOM_CONFIG.sh`, but it must be very clear that it is _not_ an official magento patch. :)

**Comment by arosenhagen** [comment]
thx for the clarification - I see your point.
As a side note: I wouldn't consider this a "official" patch anyway as long as it is not available on magento.com itself (at least for CE there is absolutely no evidence on their download page).
We ran into this very problem months ago, trying to find out why a very popular shop just hangs randomly at 100 router match iterations. I stumbled upon this repo by chance and was very in luck to have the issue resolved. I don't see, why Magento is not paying more attention to it :-/

P.S.: _"because Magento don't do that for anything but security patches for some reason"_ is a bummer since we know how they break functionality with their "security patches" (SUPEE-6788). But you're indeed not to blame for writing up such a detailed description of the issue. Keep it up :+1

**Comment by convenient** [comment]
Yeah. Sadly thats about as "official" as this is going to get. If you have an ongoing Enterprise licence agreement with Magento I believe you should be able to open a ticket with them and ask for SUPEE-4755 by name, but that's not really a scalable approach.

I'm glad to hear that this repo helped out in your instance. Sadly, there's more than one way to trigger the error and we're trying to identify and patch them as they make themselves known. 

I'll leave this issue open, once I distill my changes from `IMPROVEMENTS.md` into something more provable and testable then perhaps Magento will convert them into a new `.sh` patchfile. If that happens I'll definitely be adding it to the repository.

Thanks for this issue!
